test/Parser.unittest.js

```diff
@@ -28,7 +28,7 @@ describe("Parser", () => {
 		],
 		"call member using bracket notation": [
 			function() {
-				cde["abc"]("membertest");
+				cde.abc("membertest");
```

Comment:
Please don't change these two lines and add `// eslint-disable-next-line dot-notation`. These lines are testing if the parser can handle these things.